anyway/parsers/infographics_data_cache_updater.py

```diff
@@ -49,6 +49,7 @@ def add_news_flash_to_cache(news_flash: NewsFlash):
         if not (
             news_flash.accident
             and anyway.infographics_utils.is_news_flash_resolution_supported(news_flash)
+            and news_flash.newsflash_location_qualification is not None
```

Comment:
Hi Shai, why is this condition needed?

Reply:
there are places in the tests of the data_cache(test_infographics_data_cache.py) where a new road to the cache in added but without commit to the database, in this case no default value will be assigen to newsflash_location_qualification.
this check make sure that the user set default value in the tests.